examples/building.rs

```diff
@@ -151,11 +152,11 @@ fn place_blocks(
         if *game_mode == GameMode::Survival {
             // check if the player has the item in their inventory and remove
             // it.
-            if stack.count() > 1 {
-                let count = stack.count();
+            if stack.count > 1 {
+                let count = stack.count;
                 inventory.set_slot_amount(slot_id, count - 1);
```

Comment:
```suggestion
                inventory.set_slot_amount(slot_id, stack.count - 1);
```

Reply:
![image](https://github.com/valence-rs/valence/assets/106328960/661f5793-2bd8-40a5-af65-e1c0ededfaed)

I think this is fine
```rust
if stack.count > 1 {
    let amount = stack.count - 1;
    inventory.set_slot_amount(slot_id, amount);
} else {
    inventory.set_slot(slot_id, ItemStack::EMPTY);
}
```